app/templates/content/index.hbs

```diff
@@ -107,7 +107,9 @@
                             <div class=" p-v-xs pull-right">
                                 {{t "content.share.views"}}: {{apiMeta.metrics.views}} | {{t "content.share.downloads"}}: {{apiMeta.metrics.downloads}}
                                 {{#bs-tooltip}}
-                                    {{t "content.share.metrics_disclaimer" date=(moment-format this.metricsStartDate "YYYY-MM-DD")}}
+                                    {{t "content.share.metrics_disclaimer"}}
+                                    <br>
```

Comment:
Could you use `<br />` here?

Reply:
Huh, apparently "Self-closing void element is redundant" according to our linter. https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/self-closing-void-elements.md

Reply:
Interesting. Well, ignore me, then.